docs/advanced/github_actions.md

```diff
@@ -306,6 +306,15 @@ On self-hosted runners, you also might want to alter the default pixi install lo
 
 1. `${{ runner.temp }}\Scripts\pixi.exe` on Windows
 
+### Using the `pyproject.toml` as a manifest file for pixi.
+You can use the pyproject.toml, but it is not the default manifest file for pixi, so it's not picked up automatically, yet.
+```yaml
+- uses: prefix-dev/setup-pixi@v0.5.1
+  with:
+    manifest-path: pyproject.toml
```

Comment:
Hmm, i'm wondering whether we should make this behavior implicit. Currently it's always running with manifest-path `$PWD/pixi.toml`